web/docs/guides/with-nextjs.mdx

```diff
@@ -294,7 +294,7 @@ export default function Account({ session }) {
         .eq('id', user.id)
         .single()
 
-      if (error) {
+      if (error && status != 416) {
```

Comment:
Watching the stream right know and saw that you missed to add returning the status above. :)

Reply:
thanks @treebee!